thrift/src/test/java/com/linecorp/armeria/server/thrift/ThriftDocServicePluginTest.java

```diff
@@ -227,21 +221,22 @@ public void testNewServiceInfo() {
     public void testNewStructInfoTest() throws Exception {
         final TypeSignature string = TypeSignature.ofBase("string");
         final List<FieldInfo> fields = new ArrayList<>();
-        fields.add(new FieldInfo("boolVal", FieldRequirement.DEFAULT, TypeSignature.ofBase("bool")));
-        fields.add(new FieldInfo("byteVal", FieldRequirement.DEFAULT, TypeSignature.ofBase("i8")));
-        fields.add(new FieldInfo("i16Val", FieldRequirement.DEFAULT, TypeSignature.ofBase("i16")));
-        fields.add(new FieldInfo("i32Val", FieldRequirement.DEFAULT, TypeSignature.ofBase("i32")));
-        fields.add(new FieldInfo("i64Val", FieldRequirement.DEFAULT, TypeSignature.ofBase("i64")));
-        fields.add(new FieldInfo("doubleVal", FieldRequirement.DEFAULT, TypeSignature.ofBase("double")));
-        fields.add(new FieldInfo("stringVal", FieldRequirement.DEFAULT, string));
-        fields.add(new FieldInfo("binaryVal", FieldRequirement.DEFAULT, TypeSignature.ofBase("binary")));
-        fields.add(new FieldInfo("enumVal", FieldRequirement.DEFAULT, TypeSignature.ofNamed(FooEnum.class)));
-        fields.add(new FieldInfo("unionVal", FieldRequirement.DEFAULT, TypeSignature.ofNamed(FooUnion.class)));
-        fields.add(new FieldInfo("mapVal", FieldRequirement.DEFAULT,
-                                 TypeSignature.ofMap(string, TypeSignature.ofNamed(FooEnum.class))));
-        fields.add(new FieldInfo("setVal", FieldRequirement.DEFAULT, TypeSignature.ofSet(FooUnion.class)));
-        fields.add(new FieldInfo("listVal", FieldRequirement.DEFAULT, TypeSignature.ofList(string)));
-        fields.add(new FieldInfo("selfRef", FieldRequirement.OPTIONAL, TypeSignature.ofNamed(FooStruct.class)));
+        fields.add(new FieldInfoBuilder("boolVal", TypeSignature.ofBase("bool")).build());
```

Comment:
I'm curious if it is worth adding `FieldInfo.of(String, TypeSignature)` for this use case. What do you think?